src/clojisr/v1/r.clj

```diff
@@ -124,40 +124,40 @@
      (r (format fmt n (name package)))
      (intern *ns* ns (r ns)))))
 
-(def r== (r "`==`"))
-(def r!= (r "`!=`"))
-(def r< (r "`<`"))
-(def r> (r "`>`"))
-(def r<= (r "`<=`"))
-(def r>= (r "`>=`"))
-(def r& (r "`&`"))
-(def r&& (r "`&&`"))
-(def r| (r "`||`"))
-(def r|| (r "`||`"))
-(def r! (r "`!`"))
-(def r$ (r "`$`"))
-
-(def captured-str
-  "For the R function [str](https://www.rdocumentation.org/packages/utils/versions/3.6.1/topics/str), we capture the standard output and return the corresponding string."
-  (r "function(x) capture.output(str(x))"))
-
-(def println-captured-str (comp println-r-lines captured-str))
-
-(def str-md (comp r-lines->md captured-str))
-
-(def r** (r "`^`"))
-(def rdiv (r "`/`"))
-(def r- (r "`-`"))
+(defn- intern-r-binary [clj-op op]
```

Comment:
Actually instead of interning I would just define all the functions manually and add docstring.

Reply:
Of course. I have no idea why I thought that defn would not work....
Will change it.

Reply:
I changed it.